src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCentersList.tsx

```diff
@@ -0,0 +1,31 @@
+import Button from '@components/common/Button/Button';
+import MedCenter from '@components/MedCentersPage/MedCentersList/MedCenter/MedCenter';
+import { useTypesSelector } from '@hooks/UseTypedSelector';
+import { fetchMedCenters } from '@store/actionCreators/medCenters';
+import React, { useEffect } from 'react';
```

Comment:
react usually goes the first in the imports

Reply:
https://github.com/KLIM-nk01/virtual-ambulance/pull/26/commits/6a7aa338bd2cf39a43f0bb25411635179e962756#diff-23c5f3b8e346a5dc5889a2295907f46b739175af80e238e38c625ccb47056ed7